packages/expo-router/src/split-view/split-view.tsx

```diff
@@ -12,6 +12,9 @@ import { Slot } from '../views/Navigator';
 
 const IsWithinSplitViewContext = createContext(false);
 
+/**
+ * For full list of supported props, see [SplitViewHostProps](https://github.com/software-mansion/react-native-screens/blob/main/src/components/gamma/split-view/SplitViewHost.types.ts#L124)
```

Comment:
```suggestion
 * For full list of supported props, see [`SplitViewHostProps`](https://github.com/software-mansion/react-native-screens/blob/main/src/components/gamma/split-view/SplitViewHost.types.ts#L124)
```

Let's add an inline syntax highlight for the interface name here.